src/index.rs

```diff
@@ -17,20 +17,30 @@ use crate::Secp256k1Signer;
 #[cfg(feature = "cosmos")]
 use crate::CosmosSigner;
 
+#[cfg(feature = "aptos")]
+use crate::AptosSigner;
+
+#[cfg(feature = "aptos")]
+use crate::MultiAptosSigner;
+
 use crate::error::BundlrError;
 
 #[derive(FromPrimitive, Display, PartialEq, Eq, Debug, Clone)]
 pub enum SignerMap {
```

Comment:
I wanted to bring this up quite some time so doing it now: this type name is quite bad, it always makes me wonder what is this about until I look at the definition. Maybe this should be renamed to `SignerType` or something else. In the end, it's not a `Map` type, it's an enumeration.